src/torchmetrics/functional/text/__init__.py

```diff
@@ -27,6 +27,7 @@
 
 if _TRANSFORMERS_AUTO_AVAILABLE:
     from torchmetrics.functional.text.bert import bert_score  # noqa: F401
+    from torchmetrics.functional.text.infolm import infolm  # noqa: F401
```

Comment:
Oh yes, I was a bit perplexed with those `_TRANSFORMERS_AUTO_AVAILABLE` and `_TRANSFORMERS_AVAILABLE`. Since `transformers>=4.0`, which we use, supports auto models, I'll drop `_TRANSFORMERS_AUTO_AVAILABLE` and replace it with `_TRANSFORMERS_AVAILABLE` everywhere.